### ISSUE TITLE
Include actual terms and policies text in the app

### ISSUE THREAD

**Issue by licaon-kter** [Blockstream/green_android#181]
...currently it's just an odd checkbox that doesn't seem to mean anything

**Comment by angelix** [comment]
You mean this checkbox?

<img src="https://github.com/Blockstream/green_android/assets/631870/7f45854e-bb51-47c6-8f05-95b75d01e8c4" width="300" />

**Comment by angelix** [comment]
> No, this checkbox:
> 
> ![3b22d5bc_x7Z6kimORX-A06f9wBcYSw](https://private-user-images.githubusercontent.com/2282799/288348093-281d80af-4ea9-4023-823d-0a1ae7395deb.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE4NTMxNjIsIm5iZiI6MTcwMTg1Mjg2MiwicGF0aCI6Ii8yMjgyNzk5LzI4ODM0ODA5My0yODFkODBhZi00ZWE5LTQwMjMtODIzZC0wYTFhZTczOTVkZWIuanBnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQUlXTkpZQVg0Q1NWRUg1M0ElMkYyMDIzMTIwNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMzEyMDZUMDg1NDIyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZWE1MGNmYWIyYjE4ZTczNTBlM2ExMTdjNDNhOTBjMzlhNzEzYWUyMTY0NjA2YTJlNWI1Y2M0MmRiZjZiN2E2YSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.jA_SFPUb7yestqnmTEhGTfZGDUudIVT4A8WNLmmIKik)
> 
> PS: I don't use your locale

It's a translation issue. 

Is that correct "Sunt de acord cu Condiții de utilizare și Politică de Confidențialitate"?

**Comment by licaon-kter** [comment]
no, please look at the colours... read issue title again :)

**Comment by angelix** [comment]
> no, please look at the colours... read issue title again :)

We do a search and replace, so all links in `I agree to the Terms of Service and Privacy Policy` like `Terms of Service` & ` Privacy Policy` should be localized.

So adding the Romanian translation will fix the issue.

**Comment by licaon-kter** [comment]
> So adding the Romanian translation will fix the issue.

Yeah, but do a add a fallback to English if the translated ones are not yet available

**Comment by angelix** [comment]
Fixed in 4cd29f9cd1444b2f63738eebdc2ed6835153411a
